### ISSUE TITLE
Suggestion for ballon property?

### ISSUE THREAD

**Issue by alejandro-rios** [skydoves/Balloon#224]
**Is your feature request related to a problem?**

Not sure if this already exists but, looking for the arrow properties i didn't see something that could let me implement this

![Screen Shot 2021-08-02 at 4 21 33 PM](https://user-images.githubusercontent.com/10689052/127925780-1ac08419-262d-48da-b72f-30acefb107bd.png)

The idea would be, instead of an arrow, to be able to add a border to highlight the source object.

**Describe the solution you'd like:**

Could be a new BallonProperty, something like:
```
setShouldHighlightOrigin(Boolean) // Default false
setHighligthStrokeWidth(Int) // Border stroke width
setHighligthStrokeColor(Int) // Border color
```

**Comment by skydoves** [comment]
Hi, @alejandro-rios !
Sorry for the late reply. Is that meaning you want to use a custom view for the arrow?

**Comment by alejandro-rios** [comment]
Yes, would be nice to have that property so I can implement it like the example image.
